### PR TITLE
Beefman Cytology Tutorial Spawner

### PR DESCRIPTION
**Pull request by Horatio22**
## About The Pull Request

Tutorial spawner for a new (to us) game mechanic.

## Why It's Good For The Game
Folks can play around with cytology without having to fight for the limited space in science. Also, there are very few spawners on icebox, so this will give the ghosts something to do. Since the spawners are beefmen, temperature won't be an issue, and beginners won't die in 10 seconds.

## Changelog
:cl:
add: Adds new ghost spawner for cytology on icemoon
add: New Russian flag bodysash for beefmen 
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -3315,6 +3315,12 @@
 #include "fulp_modules\beefman_port\code\modules\mob\beefman.dm"
 #include "fulp_modules\beefman_port\code\modules\mob\beefman_clothing.dm"
 #include "fulp_modules\beefman_port\code\modules\mob\beefman_trauma.dm"
+#include "fulp_modules\jobs\job_integration.dm"
+#include "fulp_modules\jobs\brigdoc\code\brigdoc.dm"
+#include "fulp_modules\jobs\brigdoc\code\brigdoc_clothing.dm"
+#include "fulp_modules\jobs\brigdoc\code\gamemode_overwrite.dm"
+#include "fulp_modules\jobs\brigdoc\code\landmark.dm"
```
Comment (urococt) [inline tgstation.dme:3322]: Why were these lines moved?

**Comment by ATH1909** [comment]
> and beginners won't die in 10 seconds
won't they die in 10 seconds (hyperbole) due to being beefmen? or did y'all fix that.

File: fulp_modules/cytology_spawner/beefcyto.dm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	roundstart = FALSE
+	random = TRUE
+	mob_name = "Beefman Cytology Researcher"
+	mob_species = /datum/species/beefman
+	mob_type = /mob/living/carbon/human/species/beefman
+	name = "Beefman Cytology Researcher"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	short_desc = "You are a cytological researcher in a remote scientific outpost."
+	flavour_text = "You and your fellow researcher are studying cellular biology to better understand the origins of your species.Sample the subjects provided and the surrounding area for testing."
```
Comment (ATH1909) [inline fulp_modules/cytology_spawner/beefcyto.dm:12]:
```suggestion
	flavour_text = "You and your fellow researcher are studying cellular biology to better understand the origins of your species. Sample the subjects provided and the surrounding area for testing."
```

**Comment by ATH1909** [comment]
can we get a picture of the ruin

did you give them enough places to swab, etc.?

**Comment by SgtHunk** [comment]
>  and beginners won't die in 10 seconds
> won't they die in 10 seconds (hyperbole) due to being beefmen? or did y'all fix that.

Fixed, beefmen no longer bleed in room temp, but increased temperatures will. Which makes the cold a good theme for a ruin
